R/ChromBackend.R

```diff
@@ -15,21 +16,33 @@
 #' should also be provided.
 #'
 #' Through their implementation different backends can be either optimized for
-#' minimal memory requirements or performance. Each backend needs to implemet
+#' minimal memory requirements or performance. Each backend needs to implement
 #' data access methods listed in section *Backend functions:* below.
 #'
 #' And example implementation and more details and descriptions are provided
 #' in the *Creating new `ChromBackend` classes for Chromatograms* vignette.
 #'
+#' Currently available backends are:
+#'
+#' - `ChromBackendMemory`: This backend stores chromatographic data directly
+#'   in memory, making it ideal for small datasets or testing. It can be
+#'   initialized with a `data.frame` of chromatographic data via the
+#'   `chromData` parameter and a `list` of `data.frame` entries for peaks data
+#'   using the `peaksData` parameter. These data can be accessed with the
+#'   `chromData()` and `peaksData()` functions.
+#'
+#'
 #' @section Core chromatogram variables:
 #'
 #' The *core* chromatogram variables are variables (metadata) that can/should
 #' be provided by a backend. For each of these variables a value needs to be
 #' returned, if none is defined, a missing value (of the correct data type)
-#' should be returned. The names of the core variables are returned with the
-#' `chromVariables` function.
+#' should be returned. The names of the chromatogram variables in your current
+#' chromatogram object are returned with the `chromVariables()` function.
 #'
-#' For each core chromatogram variable a dedicated access method exists.
+#' For each core chromatogram variable a dedicated access method exists. In
+#' contract with the peaks data described below, a single value should be
```

Comment:
minor thing: change *contract with* -> *contrast to*

Reply:
I'm starting to think I might have dyslexia at this point